extensions/amp-date-countdown/1.0/base-element.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DateCountdown} from './component';
+import {PreactBaseElement} from '../../../src/preact/base-element';
+import {parseDateAttrs as parseDateAttrsBase} from '../../../src/utils/date';
+
+export class BaseElement extends PreactBaseElement {}
```

Comment:
Wondering what would the `render` interface be like in CE mode without the mustache template?

Reply:
I'll be honest, I'm not familiar enough with the components to to know how to answer this